kolibri/core/assets/src/components.js

```diff
@@ -95,6 +95,89 @@ var Tag = Mn.ItemView.extend({
     }
 });
 
+
+// Implements CRUD actions for a given item
+var CrudItem = Mn.ItemView.extend({
```

Comment:
Yes, however I think it bears a little more design scrutiny before being used for real.